3.7/README.md

```diff
@@ -43,7 +43,7 @@ in Openshift can be achieved with the following step:
 The same application can also be built using the standalone [S2I](https://github.com/openshift/source-to-image) application on systems that have it available:
 
     ```
-    $ s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=3.7/test/setup-test-app/ rhscl/python-37-rhel7 python-sample-app
+    $ s2i build https://github.com/sclorg/s2i-python-container.git --context-dir=3.7/test/setup-test-app/ <image_name> python-sample-app
```

Comment:
The same here.

Reply:
This file is generated via distgen so we don't have to discuss this change for every single file.

Reply:
BTW, the example is not valid and does not work. We don't have Python 3.7 on RHEL 7/8 and we never will. That's the main reason I am changing this.